lib/components/Accordion/index.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+    ReactNode,
+    memo,
+    useCallback,
+    useEffect,
+    useRef,
+    useState,
+} from "react";
+import clsx from "clsx";
+import Chevron from "./Chevron.svg";
+import "./Accordion.scss";
+
+type AccordionVariants = "underline" | "bordered" | "shadow";
+
+type AccordionSection = {
+    title: ReactNode;
```

Comment:
```suggestion
    id: string;
    title: ReactNode;
```
I'm suggesting this in order to not use index number to detect the active section